apps/browser/src/_locales/en/messages.json

```
@@ -3087,7 +3087,7 @@
     "message": "Invalid file password, please use the password you entered when you created the export file."
   },
   "importDestination": {
```

Any change of message content without changing the key won't be reflected in existing translations. Please change the key and replace it's usages. This needs to be done for all clients.
```suggestion
  "destination": {
```